bot/handlers/sending_videos/compile_clips_handler.py

```diff
@@ -33,15 +33,22 @@
 
 
 class CompileClipsHandler(BotMessageHandler):
+
     class ParseSegmentsException(Exception):
+        """Exception raised during segment parsing."""
         def __init__(self, message: str) -> None:
             self.message = message
             super().__init__(self.message)
```

Comment:
```suggestion
```

Powinno bez tego śmigać, zwłaszcza że nic tutaj nie robisz ponad standardową logikę i chcesz to tylko nazwać